src/Services/Utils.php

```diff
@@ -40,24 +40,28 @@ public function __construct(ManagerRegistry $doctrine, TranslatorInterface $tran
         $this->doctrine = $doctrine;
     }
 
-    /**
-     * Hash a password according to the realm.
-     * Important note: It is very insecure and this is used only for the legacy sabre/dav implementation.
-     */
-    public function hashPassword(string $username, string $password): string
```

Comment:
As indicated in the comment, this function is needed for legacy sabre/dav implementations

Reply:
Understood, I missed the note as I was breezing through at the end of the day when I made the commit.